api/v1beta1/foundationdbbackup_types.go

```diff
@@ -67,14 +70,16 @@ type FoundationDBBackupSpec struct {
 
 	// The name for the backup.
 	// If empty defaults to .metadata.name.
+	// Deprecated use BlobStoreConfiguration instead
 	BackupName string `json:"backupName,omitempty"`
 
 	// The account name to use with the backup destination.
-	// +kubebuilder:validation:Required
-	AccountName string `json:"accountName"`
```

Comment:
Currently we don't have a deprecation-checking mechanism, I would post-pone this until w have the backup CRD/setup in a more stable state.

Reply:
That sounds reasonable. I'll file a follow-up issue on that. We should probably avoid removing any deprecated fields on the backup and restore resources until we have that checker in place, to avoid catching people by surprise.